### PR TITLE
Update PR template

### PR DESCRIPTION
**Pull request by Vincevrp**
### Description

* Replace HTML preview with screenshots since this doesn't work with Jekyll
* Reference contributing guidelines
* Add place to fill in issue number

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 
+Issue: # (if applicable)
 
-### HTML Preview
+## Screenshots
 
-*Replace [GITHUB_USERNAME] with your GitHub username and [BRANCH] with the branch name.*
-
-https://htmlpreview.github.io/?https://github.com/[GITHUB_USERNAME]/privacytools.io/blob/[BRANCH]/index.html
+Please add screenshots if applicable
```
Comment (Shifterovich) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would make this an HTML comment. People who open PRs will see that but it won't be rendered in the markdown.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-### Description
+**PLEASE READ OUR [CONTRIBUTING GUIDELINES](https://github.com/privacytoolsIO/privacytools.io#contributing-guidelines) BEFORE SUBMITTING**
 
+## Description
 
+Issue: # (if applicable)
```
Comment (Shifterovich) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe add `(none)` as a default value?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
-### Description
+**PLEASE READ OUR [CONTRIBUTING GUIDELINES](https://github.com/privacytoolsIO/privacytools.io#contributing-guidelines) BEFORE SUBMITTING**
```
Comment (Shifterovich) [inline .github/PULL_REQUEST_TEMPLATE.md]: Make this an HTML comment.

**Comment by Shifterovich** [comment]
Thanks!

**Comment by Shifterovich** [comment]
https://github.com/laravel/framework/issues/new/choose I like this.

Would be cool if we could implement that. Like "Software suggestion", "Software removal", "Feature", "Discussion" or some such categories.

[Here's the code]( https://github.com/laravel/framework/tree/5.7/.github/ISSUE_TEMPLATE) laravel/framework uses for this.

GH documentation: https://blog.github.com/2018-05-02-issue-template-improvements/

Could you try adding something like that?

**Comment by Vincevrp** [comment]
> https://github.com/laravel/framework/issues/new/choose I like this.
> 
> Would be cool if we could implement that. Like "Software suggestion", "Software removal", "Feature", "Discussion" or some such categories.
> 
> [Here's the code](https://github.com/laravel/framework/tree/5.7/.github/ISSUE_TEMPLATE) laravel/framework uses for this.
> 
> GH documentation: https://blog.github.com/2018-05-02-issue-template-improvements/
> 
> Could you try adding something like that?

Looks really good. I'll have a look at this!

**Pull request merged**